presto-common/src/main/java/com/facebook/presto/common/type/UnknownType.java

```diff
@@ -130,7 +132,7 @@ public boolean getBoolean(Block block, int position)
     {
         // Ideally, this function should never be invoked for unknown type.
         // However, some logic rely on having a default value before the null check.
-        checkArgument(block.isNull(position));
+        checkArgument(block.isNull(position), "This function should never be called");
```

Comment:
The message, according to the check argument, should be "position %d is null in block". If this function (`getFixedSize`) should never be called, we should just throw `UnsupportedOperationException`. Otherwise the message is very confusing.

Reply:
Done